src/docs/markdown/caddyfile/options.md

```diff
@@ -295,6 +295,7 @@ Skips the attempts to install the local CA's root into the system trust store, a
 ##### `acme_ca`
 Specifies the URL to the ACME CA's directory. It is strongly recommended to set this to Let's Encrypt's [staging endpoint](https://letsencrypt.org/docs/staging-environment/) for testing or development. Default: ZeroSSL and Let's Encrypt's production endpoints.
 
+Note that the [hostname requirements](/docs/automatic-https#hostname-requirements) apply, and ACME CA directory will not be used for IP addresses, `localhost` and hostnames with a `.local` TLD.
```

Comment:
```suggestion
Note that a globally-configured ACME CA may not apply to all sites; see the [hostname requirements](/docs/automatic-https#hostname-requirements) for using the default ACME issuer(s).
```

What about something like this? The sentence can remain but we link to the relevant docs instead of duplicating them.

Reply:
Yeah I’m good with anything, it’s your project ^^